pyproject.toml

```diff
@@ -72,6 +72,11 @@ classifiers = [
 subprocess = [
   "subprocess32>=3.2.7; python_version < '3'"
 ]
+management = [
+  # N.B.: Released on 2017-09-01 and added support for the `process_iter(attrs, ad_value)` API we
+  # use in `pex.cache.access`.
+  "psutil>=5.3"
```

Comment:
N.B.: If Pants wants to use these cache management commands and it still uses the Pex PEX releases, I'll either need to add a `pex+management` PEX to the release that embeds psutil for all supported platforms or else start releasing Pex PEX scies, which naturally handle platform specific deps. I favor the scies here since the pex+management approach doesn't scale well once more CLI-specific-deps are added, but I wanted to get Pants maintainers opinions on this.

Reply:
... there's a longer story on why I went with psutil / access info gathering at delete-time, but suffice it to say, recording usage info in the read-write access lock path (I used sqlite3) added ~2-30ms overhead to every PEX launch and I did not deem that acceptable.

Reply:
Well, two reasons:
1. It means embedding all these wheels in the Pex PEX or the Pex+management PEX - so, in part size:
    + psutil-5.9.5-cp27-cp27m-macosx_10_9_x86_64.whl
    + psutil-5.9.5-cp27-cp27m-manylinux2010_i686.whl
    + psutil-5.9.5-cp27-cp27m-manylinux2010_x86_64.whl
    + psutil-5.9.5-cp27-cp27mu-manylinux2010_i686.whl
    + psutil-5.9.5-cp27-cp27mu-manylinux2010_x86_64.whl
    + psutil-5.9.5-cp36-abi3-macosx_10_9_x86_64.whl
    + psutil-5.9.5-cp36-abi3-manylinux_2_12_i686.manylinux2010_i686.manylinux_2_17_i686.manylinux2014_i686.whl
    + psutil-5.9.5-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl
    + psutil-5.9.5-cp38-abi3-macosx_11_0_arm64.whl
2. But ut means inventing a bit of new functionality to support a `Universal` `Target` to complement the current `AbbreviatedPlatform`, `CompletePLatform` and `LocalInterpreter` `Target`s - this new `Target` type would only work when the Pex resolve was against a --lock or a --pex-repository and it would be able to grab all the wheels listed in 1 despite a machine not having all the corresponding `LocalInterpreter` targets available.

Reply:
Hrm, I think I need to do most of 2 anyhow to support creating a lock for the Pex scies with something like `pex3 lock create --project ".[management]" --pip-version latest --style universal --target-system linux --target-system mac --interpreter-constraint "CPython==3.12.*" -o pex-scie.lock`. Alternatively, I could create a manual workflow that generated a complete platform for each of the 4 supported platforms using the scie PBS and generate a strict multi-lock using those 4 `--complete-platform` targets.

Reply:
If that's a hassle then an alternative is for Pants to install pex as a dist, with the management extra, when it needs to run management commands?

Reply:
Aha, OK. Yeah - that would work. I should have the Pex scies + release PRs up today.